resources/cmake/FindRTIConnextDDS.cmake

```diff
@@ -264,6 +264,31 @@
 # - All the Darwin platforms (OS X 10.11-10.13)
 #
 
+#####################################################################
+# Logging Functions                                                  #
+#####################################################################
+function(connextdds_debug message)
+    message(DEBUG ${message})
```

Comment:
According to the documentation the DEBUG and VERBOSE levels were introduced in 3.15. Have we tried the behavior on previous versions? Were just undocumented levels?